elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchBackendQueueProcessor.java

```diff
@@ -24,10 +35,12 @@
 public class ElasticsearchBackendQueueProcessor implements BackendQueueProcessor {
 
 	private ElasticsearchIndexManager indexManager;
+	private ExtendedSearchIntegrator searchIntegrator;
 
 	@Override
 	public void initialize(Properties props, WorkerBuildContext context, IndexManager indexManager) {
 		this.indexManager = (ElasticsearchIndexManager) indexManager;
+		this.searchIntegrator = context.getUninitializedSearchIntegrator();
```

Comment:
The SI in the index manager is still null at this point, so this doesn't work.

But I can take the SI from the context and initialize the visitor here with it once. At the time it will acutally be used, the SI should be fully set up. I've filed PR #1022 for this.